src/components/program-progress/tests/ProgramListingCard.test.jsx

```diff
@@ -118,20 +98,46 @@ describe('<ProgramListingCard />', () => {
       key: 'test-key2',
       logoImageUrl: '/media/organization/logos/shield.png',
     });
-    const { queryByAltText } = renderWithRouter(<ProgramListingCardWithContext
-      initialUserSubsidyState={userSubsidyState}
-      programData={dummyDataWithMultipleOrg}
-    />);
+    const { queryByAltText } = renderWithRouter(
+      <ProgramListingCardWithContext programData={dummyDataWithMultipleOrg} />,
+    );
     const logoImageNode = queryByAltText(dummyProgramData.authoringOrganizations[0].key);
     expect(logoImageNode).toBeNull();
   });
 
   it('redirects to correct page when clicked', () => {
-    const { container } = renderWithRouter(<ProgramListingCardWithContext
-      initialUserSubsidyState={userSubsidyState}
-      programData={dummyProgramData}
-    />);
+    const { container } = renderWithRouter(<ProgramListingCardWithContext programData={dummyProgramData} />);
     userEvent.click(container.firstElementChild);
     expect(window.location.pathname).toEqual(`/${mockEnterpriseCustomer.slug}/program/test-uuid/progress`);
   });
+
+  it.each([{
+    width: 1450,
+    size: 'large',
+  },
+  {
+    width: 1300,
+    size: 'large',
+  },
+  {
+    width: 1000,
+    size: 'large',
+  },
+  {
+    width: 800,
+    size: 'medium',
+  },
+  {
+    width: 600,
+    size: 'small',
+  },
+  {
+    width: 500,
+    size: 'xSmall',
+  }])('tests window size', ({ width, size }) => {
+    global.innerWidth = width;
+    renderWithRouter(<ProgramListingCardWithContext programData={dummyProgramData} />);
+    const imageCapSrc = screen.getByAltText('Program banner image').src;
```

Comment:
Ah, my comment about passing empty string to `alt` here (because the image is purely decorative) would break this test. Would there be another way to get at the image src here?

Reply:
I'll try to find another way to reference to element, but that was the primary reason why the srcAlt was added since the data-testid was not being passed through the imageCap component